opentelemetry-integration-example/src/opentelemetry_integration_example/flask_example.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+This module serves as an example to integrate with flask, using
+the requests library to perform downstream requests
+"""
+import flask
+import requests
+from opentelemetry import trace
+import opentelemetry.ext.http_requests
+from opentelemetry.sdk.trace import Tracer
+from opentelemetry.ext.wsgi import OpenTelemetryMiddleware
+
+
+def configure_opentelemetry(app: flask.Flask):
+    """Configure a flask application to use OpenTelemetry.
+
+    This activates the specific components:
+
+    * sets tracer to the SDK's Tracer
+    * enables requests integration on the Tracer
+    * uses a WSGI middleware to enable configuration
+
+    TODO:
+
+    * exporters?
+    * configure propagators / formatters
```

Comment:
Curious, is formatters the same as processors?

Reply:
no, I think I misread the spec an have been calling it formatters when I meant propagators: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-propagators.md

I think there's some terminology that could be clarified there, but I'll align with the correct terminology for now.